beacon-chain/rpc/beacon/committees.go

```diff
@@ -105,6 +105,37 @@ func (bs *Server) retrieveCommitteesForEpoch(
 	return committeesListsBySlot, activeIndices, nil
 }
 
+func (bs *Server) retrieveCommitteesForRoot(
```

Comment:
On second thought, can you add a comment for `retrieveCommitteesForRoot` that says this is preferred to use than `retrieveCommitteesForEpoch`, because retrieving a state by root is more performant than retrieving a state by slot...etc

Thanks!